akka-http-core/src/main/scala/akka/http/impl/settings/RoutingSettingsImpl.scala

```diff
@@ -28,6 +30,6 @@ object RoutingSettingsImpl extends SettingsCompanion[RoutingSettingsImpl]("akka.
     c getInt "range-count-limit",
     c getBytes "range-coalescing-threshold",
     c getIntBytes "decode-max-bytes-per-chunk",
-    c getString "file-io-dispatcher")
-
-}
+    c getString "file-io-dispatcher",
+    ErrorLoggingVerbosity(c getString "error-logging-verbosity"))
```

Comment:
I don't think we can remove `ErrorLoggingVerbosity` from `ParserSettings`, because of `Public API but not intended for subclassing`. We could copy it into `RoutingSettings`, but it is kind of meh. We also could create similar object in `RoutingSettings`, but we cannot remove or move current `ErrorLoggingVerbosity` due to `*Settings` being public.